tensorflow_lite_support/ios/ios.bzl

```diff
@@ -2,29 +2,38 @@
 
 # When the static framework is built with bazel, the all header files are moved
 # to the "Headers" directory with no header path prefixes. This auxiliary rule
-# is used for stripping the path prefix to the C API header files included by
-# other C API header files.
-def strip_c_api_include_path_prefix(name, hdr_labels, prefix = ""):
-    """Create modified header files with the common.h include path stripped out.
+# is used for stripping the path prefix to the C/iOS API header files included by
+# other C/iOS API header files.
+# In case of C header files includes start with a keyword of "#include'.
+# Imports in iOS header files start with a keyword of '#import'.
+def strip_api_include_path_prefix(name, hdr_labels, prefix = ""):
+    """Create modified header files with the import path stripped out.
 
     Args:
       name: The name to be used as a prefix to the generated genrules.
       hdr_labels: List of header labels to strip out the include path. Each
           label must end with a colon followed by the header file name.
       prefix: Optional prefix path to prepend to the header inclusion path.
     """
-
     for hdr_label in hdr_labels:
         hdr_filename = hdr_label.split(":")[-1]
-        hdr_basename = hdr_filename.split(".")[0]
+        import_keyword = "#include"
+        # The last path component of iOS header files can be sources/some_file.h 
+        # or Sources/some_file.h. Hence it wiill contain a '/'. So we can safely 
+        # assume that the import keyword will be #import
+        if "/" in hdr_filename:
+            hdr_filename = hdr_filename.split("/")[-1]
+            import_keyword = "#import"
 
+        hdr_basename = hdr_filename.split(".")[0]
         native.genrule(
             name = "{}_{}".format(name, hdr_basename),
             srcs = [hdr_label],
             outs = [hdr_filename],
             cmd = """
-            sed 's|#include ".*/\\([^/]\\{{1,\\}}\\.h\\)"|#include "{}\\1"|'\
+            sed 's|{} ".*/\\([^/]\\{{1,\\}}\\.h\\)"|{} "{}\\1"|'\
```

Comment:
@yyoon I am a little unfamiliar with regex. There may be a better way to do this. Please take a look.